pkgs/development/python-modules/nltk/default.nix

```diff
@@ -1,15 +1,16 @@
-{ fetchurl, buildPythonPackage, lib, six, pythonAtLeast, pythonOlder }:
+{ fetchurl, buildPythonPackage, lib, six, pythonAtLeast, pythonOlder
+, singledispatch }:
 
 buildPythonPackage rec {
-  version = "3.2.5";
+  version = "3.4";
   pname = "nltk";
 
   src = fetchurl {
-    url = "mirror://pypi/n/nltk/nltk-${version}.tar.gz";
-    sha256 = "2661f9971d983db314bbebd51ba770811a362c6597fd0f303bb1d3beadcb4834";
+    url = "mirror://pypi/n/nltk/nltk-${version}.zip";
+    sha256 = "286f6797204ffdb52525a1d21ec0a221ec68b8e3fa4f2d25f412ac8e63c70e8d";
   };
 
-  propagatedBuildInputs = [ six ];
+  propagatedBuildInputs = [ singledispatch six ];
 
   disabled = pythonOlder "2.7" || pythonOlder "3.4" && (pythonAtLeast "3.0");
```

Comment:
This is no longer needed.

Reply:
Partly true. we need to patch `setup.py` to remove this dependency for python3.4+ 
https://github.com/nltk/nltk/blob/3.4/setup.py#L98

Reply:
I was referring to `disabled = ...` line, which you removed :+1: